<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_game_settings</class>
 <widget class="QDialog" name="new_game_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="new_game_settings_buttonbox">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>100</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="new_game_settings_layout">
    <item>
     <layout class="QVBoxLayout" name="spieler_layout">
      <item>
       <widget class="QRadioButton" name="two_player_btn">
        <property name="text">
         <string>2 Spieler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="three_player_btn">
        <property name="text">
         <string>3 Spieler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="four_player_btn">
        <property name="text">
         <string>4 Spieler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="spielfeld_layout">
      <item>
       <widget class="QLabel" name="spielfeld_label">
        <property name="text">
         <string>Spielfeld</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hoehe_layout">
        <item>
         <widget class="QLabel" name="hoehe_label">
          <property name="text">
           <string>HÃ¶he</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hoehe_spinbox">
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="breite_layout">
        <item>
         <widget class="QLabel" name="breite_label">
          <property name="text">
           <string>Breite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="breite_spinbox">
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Spielernamen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>Guybrush</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>Dr. Fred</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="text">
       <string>Purpur</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>LeChuck</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>new_game_settings_buttonbox</sender>
   <signal>accepted()</signal>
   <receiver>new_game_settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>new_game_settings_buttonbox</sender>
   <signal>rejected()</signal>
   <receiver>new_game_settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
